packages/graphql/src/server.js

```diff
@@ -13,7 +13,7 @@ try {
   console.warn('EnvKey not configured')
 }
 
-setNetwork(process.env.NETWORK || 'test')
+setNetwork(process.env.NETWORK || 'test', { performanceMode: false })
```

Comment:
Is the performanceMode config used in any ways by the graphQL server ?
Or is this just defensive coding for future ?

Reply:
This was completely defensive, and a little unnecessary.  No, it should never actually come into play.  I could probably remove this if you want, but it made me feel better.

Reply:
I'm happy to have it here!

Reply:
Sounds good - thanks for explaining.